queries.sql

```diff
@@ -34,4 +34,29 @@ SELECT COUNT(*) FROM animals WHERE escape_attempts < 1;
 SELECT AVG(weight_kg) FROM animals;
 SELECT neutered, MAX(escape_attempts) FROM animals GROUP BY neutered;
 SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
-SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-01-01' GROUP BY species;
\ No newline at end of file
+SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-01-01' GROUP BY species;
+
+CREATE TABLE owners(
+  id SERIAL PRIMARY KEY,
+  full_name VARCHAR(255),
+  age INT
+);
+
+CREATE TABLE
+  CREATE TABLE species(
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(255)
+);
+
+ALTER TABLE animals DROP COLUMN species;
+ALTER TABLE animals ADD COLUMN species_id INT, ADD CONSTRAINT species_id FOREIGN KEY(species_id) REFERENCES species(id);
+ALTER TABLE animals ADD COLUMN owners_id INT, ADD CONSTRAINT owners_id FOREIGN KEY(owners_id) REFERENCES owners(id);
```

Comment:
- [ ] These are not supposed to be here. KIndly place them in the right file as per the project requirements.